doc/setup/SETUP.md

```diff
@@ -92,7 +92,7 @@ info:
     imprint: https://ase.in.tum.de/lehrstuhl_1/component/content/article/179-imprint
 ```
 Change all entries with ```<...>``` with proper values, e.g. your TUM Online account credentials to connect to the given instances of JIRA, Bitbucket and Bamboo. Alternatively, you can connect to your local JIRA, Bitbucket and Bamboo instances.
-Be careful that you don't commit changes in this file. Best practice is to specify that your local git repository ignores this file or assumes that this file is unchanged.
+Be careful that you don't commit changes in this file. Best practice is to specify that your local git repository ignores this file or assumes that this file is unchanged. Also note that except for the credentials and a random encryption-password, other fields like `ldap` can be left blank.
```

Comment:
You do not even need the credentials and the default encryption-password also works. 😋

Reply:
you might want to reference https://github.com/ls1intum/Artemis/blob/develop/doc/jenkins-gitlab/JENKINS_AND_GITLAB_SETUP.md and https://github.com/ls1intum/Artemis/blob/develop/doc/bamboo-bitbucket-jira/BAMBOO_BITBUCKET_JIRA_SETUP.md for more details about the yml config

Reply:
Done